src/app/frontend/replicasetdetail/replicasetdetail.scss

```diff
@@ -84,6 +84,11 @@ $replicasetdetails-table-cell: #777777;
   font-size: 0.85em;
 }
 
+.kd-replicasetdetail-sidebar-button {
```

Comment:
Unfortunately it still requires removing some padding from button to make it look good, but  at least it looks like a button now.